File: SymulatorSwiataCpp/SymulatorSwiata/Owca.cpp
#include "Owca.h"
#include "zasoby.h"

Owca::Owca(int x, int y)
{
	this->wiek = 0;
	this->x = x;
	this->y = y;
	this->wyglad = OWCA;
	this->inicjatywa = 4;
	this->sila = 4;
}

void Owca::akcja(Swiat &swiat, int i, int j)
{
	nowe = losuj();
	this->kolizja(swiat, i, j);
	wiek++;
}

File: SymulatorSwiataJava/src/Zwierzeta/Owca.java
package Zwierzeta;
import java.awt.Color;

import Organizmy.*;

public class Owca extends Zwierze {

	private Zasoby a;
	@Override
	public void akcja(Swiat swiat, int i, int j) {
		nowe = losuj();
		kolizja(swiat, i, j);
		wiek++;
	}
	
	public Owca(int x1, int y1)
	{
		wiek = 0;
		x = x1;
		y = y1;
		wyglad = 'O';
		inicjatywa = 4;
		sila = 4;
		color = Color.CYAN;
	}

}
